### ISSUE TITLE
opnReqTest  does not work with cpld_bmark_p7

### ISSUE THREAD

**Issue by JessicaMeixner-NOAA** [ufs-community/ufs-weather-model#899]
## Description
The opnReqTest  does not work with cpld_bmark_p7

### To Reproduce:

from test directory, run: 

./opnReqTest -n cpld_bmark_p7 -k -c mpi,thr,dcp

## Additional context
Directly reference any issues or PRs in this or other repositories that this is related to, and describe how they are related. Example:
- needs to be fixed also in noaa-emc/nems/issues/<issue_number>
- needed for noaa-emc/fv3atm/pull/<pr_number>

## Output

```
./opnReqTest -n cpld_bmark_p7 -k -c mpi,thr,dcp
hfe11
Machine:  hera.intel     Account:  nems
test name: cpld_bmark_p7
cases to compile: std
compiling std with compile option -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v16_coupled_nsstNoahmpUGWPv1
done compiling std
cases to run: std_base mpi thr dcp
Running test for std_base with
    THRD: 1; INPES: 6; JNPES: 8; TASKS: 560; TPN: 40
Coupled application not yet implemented for mpi
Died with error code 1
```

**Comment by MinsukJi-NOAA** [comment]
@JessicaMeixner-NOAA can you try without mpi? `./opnReqTest -n cpld_bmark_p7 -k -c thr,dcp`. Since you must have the executable already created, you can add `-x` flag as well to skip compilation

**Comment by junwang-noaa** [comment]
@JessicaMeixner-NOAA <https://github.com/JessicaMeixner-NOAA>
cpld_bmark_p7 has the wave component, we don't expect it will work with dcp
either. Are you running the test for debugging?

**Comment by JessicaMeixner-NOAA** [comment]
Trying to be proactive in showing that WW3 can pass more of these tests now with some upcoming code changes. I have also added new tests, but had added those before the opnReqTest was here to use so wanted to try this as well as extra tests might not be wanted in the repository. 

I'll try just: 
`  ./opnReqTest -n cpld_bmark_p7 -k -c thr
`
and report back

**Comment by JessicaMeixner-NOAA** [comment]
@MinsukJi-NOAA 
```
$ ./opnReqTest -n cpld_bmark_p7 -k -c thr,dcp
hecflow01
Machine:  hera.intel     Account:  nems
test name: cpld_bmark_p7
cases to compile: std
compiling std with compile option -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v16_coupled_nsstNoahmpUGWPv1
done compiling std
cases to run: std_base thr dcp
Running test for std_base with
    THRD: 1; INPES: 6; JNPES: 8; TASKS: 560; TPN: 40
Running test for thr with
    THRD: 2; INPES: 3; JNPES: 4; TASKS: 138; TPN: 20
Died with error code 1
```

Should I try cpld_control_wave_p7 instead? Or just cpld_control_p7 works?

**Comment by MinsukJi-NOAA** [comment]
> @MinsukJi-NOAA
> 
> ```
> $ ./opnReqTest -n cpld_bmark_p7 -k -c thr,dcp
> hecflow01
> Machine:  hera.intel     Account:  nems
> test name: cpld_bmark_p7
> cases to compile: std
> compiling std with compile option -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v16_coupled_nsstNoahmpUGWPv1
> done compiling std
> cases to run: std_base thr dcp
> Running test for std_base with
>     THRD: 1; INPES: 6; JNPES: 8; TASKS: 560; TPN: 40
> Running test for thr with
>     THRD: 2; INPES: 3; JNPES: 4; TASKS: 138; TPN: 20
> Died with error code 1
> ```
> 
> Should I try cpld_control_wave_p7 instead? Or just cpld_control_p7 works?

@JessicaMeixner-NOAA 
The number of tasks does not look right for `thr`. It may have been hard coded for cpld_control_p7. Can you please try `cpld_control_wave_p7`, which I believe should have the same tile layout as `cpld_control_p7`? If that does not work, I will work on making `cpld_control_wave_p7` work with opnReqTest script; I am not sure if the benchmark test is a good candidate for the opnReqTest script.

**Comment by JessicaMeixner-NOAA** [comment]
Looks like the cpld_control_wave_p7 also does not work: 

```
./opnReqTest -n cpld_control_wave_p7 -k -c thr
hecflow01
Machine:  hera.intel     Account:  nems
test name: cpld_control_wave_p7
cases to compile: std
compiling std with compile option -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v16_coupled_nsstNoahmpUGWPv1
done compiling std
cases to run: std_base thr
Running test for std_base with
    THRD: 1; INPES: 3; JNPES: 8; TASKS: 200; TPN: 40
Running test for thr with
    THRD: 2; INPES: 3; JNPES: 4; TASKS: 120; TPN: 20
Died with error code 1
```

I'm curious as to why the benchmark test would not be a good idea?  Sometimes the components have features that are resolution dependent, so I would think having the capability to run any resolution might be important.

**Comment by MinsukJi-NOAA** [comment]
I agree that different resolutions could be important. However, I am not sure if the operation test should deal with the high number of PE's used by benchmark to meet forecast time requirement..

**Comment by JessicaMeixner-NOAA** [comment]
So is the recommendation then that we stick to manual testing of the higher resolution test cases to ensure the resolutions meet the operational requirements?   

Here are two cases where resolution has an impact (there are likely more, and likely other reasons and arguments for just testing the lower): 
* Ocean physics is different at the 1 deg versus 1/4 deg resolution 
* Currently, the wave model passes the MPI test at the lower resolution, but not at the higher resolution which exposed an issue.  So, in this case, the wave model would appear to pass the operational requirements, but for operational resolutions, it would in fact not pass.

**Comment by junwang-noaa** [comment]
Thanks, I think the opnReqTest needs to support high resolution tests too, e,g, the cpld_bmark cases.  But please keep in mind that the developer won't be able to run those tests or need to require a special queue if the high resolution tests require a large amount of nodes that developers may not be able to get access to.

**Comment by MinsukJi-NOAA** [comment]
> I agree that different resolutions could be important. However, I am not sure if the operation test should deal with the high number of PE's used by benchmark to meet forecast time requirement..

@JessicaMeixner-NOAA Maybe my previous comment was not clear.  I agree with you on testing different resolutions. My uncertainty was in regards to using the benchmark test; here is an example. If `cpld_control_p7` and `cpld_bmark_p7` have the same resolutions but different number of PETs (since benchmark needs to run faster to meet the run time requirement), should we test operation requirement on `cpld_control_p7`?

**Comment by JessicaMeixner-NOAA** [comment]
The cpld_control_p7 is a C96/1 deg test, whereas cpld_bmark_p7 is C384/0.25 deg + waves test, so I think the different resource requirements between the two is due to the resolution differences.  The cpld_control_c384_p7 and cpld_bmark_p7 have the same atm/ocn resolution but the bmark test also includes waves.

**Comment by MinsukJi-NOAA** [comment]
ORT changes for `cpld_bmark_p7` has been merged via #931
